app/src/main/java/com/gholem/moneylab/features/chart/adapter/item/ChartItem.kt

```diff
@@ -1,15 +1,21 @@
 package com.gholem.moneylab.features.chart.adapter.item
 
+import com.gholem.moneylab.domain.model.TransactionModel
+
 sealed class ChartItem {
     data class Category(
-        val categpry: ChartCategoryModel
+        val categpry: TransactionModel
     ) : ChartItem()
 
     data class Pie(
-        val transactionModel: List<ChartCategoryModel>
+        val transactionModel: List<TransactionModel>
     ) : ChartItem()
 
     data class Bar(
-        val transactionModel: List<ChartCategoryModel>
+        val transactionModel: List<TransactionModel>
+    ) : ChartItem()
+
+    data class Retrofit(
```

Comment:
I will suggest to change the name to just `Transaction` - this represent more of what it is. The source of data completely doesn't matter here and shouldn't be included in the name.